### ISSUE TITLE
progress_bar not displayed

### ISSUE THREAD

**Issue by ElisonSherton** [fastai/fastprogress#62]
Hi there!

I installed fastprogress using pip. I am trying to use this in order to create a progressbar for the sake of measuring time required to parse all the documents using spacy. 

The issue is that the progressbar is not being displayed. Can you please help me with the same? I have tried to create a toy example to demonstrate the same below.

![image](https://user-images.githubusercontent.com/45599762/80956947-d3658e80-8e1f-11ea-8de1-c128066df552.png)

As seen above, there's a small progressbar with no comment and it's static all the while.

Can you please help me with this?

With Regards,
Vinayak.

**Comment by tcapelle** [comment]
put a sleep(0.1) inside probably this is instant.

**Comment by import-antigravity** [comment]
I'm having a similar problem. Running some of the example code in an iPython shell in iTerm looks correct:
![Screen Shot 2020-07-17 at 11 59 30 AM](https://user-images.githubusercontent.com/24441495/87806544-0ca97a00-c825-11ea-93ba-34ac3b8b1b80.png)

But running it in pycharm (even in an iPython terminal) looks like OP's screenshot.

**Comment by yuanjinghui** [comment]
I'm having a similar problem using Pycharm. I've posted here https://discourse.pymc.io/t/pycharm-cannot-show-progress-bar-while-jupyter-notebook-shows-well-after-upgrade-the-pymc3-to-3-9-3/6090

**Comment by ohmeow** [comment]
Any resolution to this? I'm having the same issue in jupyter notebook

**Comment by fohria** [comment]
I see the same behaviour in Atom with Hydrogen plugin, but solution in #61 worked for the simple example given by @ElisonSherton above.

**Comment by ohmeow** [comment]
What's funny is that once I did an editable install, it worked just fine.
Even funnier, Zach Mueller said  nothing was different between what is on
pypi and what you get from doing an editable install ... so color me
confused still :)

**Comment by leiyu-thunder** [comment]
I have the same problem in jupyter notebook. I solved it by upgrade IPython package >= 6.0.0, which is required by fastprogress. Maybe you can try it.

**Comment by johan12345** [comment]
This still is an issue for me in PyCharm. A workaround seems to be to add the following:
```python
from fastprogress import fastprogress
fastprogress.printing = lambda: True
```
I pushed a possible fix in PR #89.

**Comment by redreddragon** [comment]
I have resolved this problem in PyCharm for Windows.

```python
from fastprogress.fastprogress import ConsoleProgressBar,ConsoleMasterBar

from time import sleep
mb = ConsoleMasterBar(range(10))
pb = ConsoleProgressBar(range(100), parent=mb)

#  pd.end is '\r' as default,  which can not be displayed in Windows
pb.end = '\r\n'

for i in mb:
    for j in pb:
        sleep(0.01)
```

More details here https://stackoverflow.com/questions/15433188/what-is-the-difference-between-r-n-r-and-n
